File: backend/src/main/java/com/pgvaale/backend/dto/TiffinDashboardDTO.java
package com.pgvaale.backend.dto;

import lombok.*;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class TiffinDashboardDTO {
    private String tiffinName;
    private Long pendingRequests;
    private Long acceptedRequests;
    private Long rejectedRequests;
    private Double averageRating;
    private List<UserTiffinDTO> recentRequests;
} 

File: PGVaaleDotNetBackend/DTOs/TiffinDashboardDTO.cs
using PGVaaleDotNetBackend.DTOs;

namespace PGVaaleDotNetBackend.DTOs
{
    public class TiffinDashboardDTO
    {
        public string TiffinName { get; set; } = string.Empty;
        public long PendingRequests { get; set; }
        public long AcceptedRequests { get; set; }
        public long RejectedRequests { get; set; }
        public double AverageRating { get; set; }
        public List<UserTiffinDTO> RecentRequests { get; set; } = new List<UserTiffinDTO>();
    }
}
